src/module.c

```diff
@@ -6065,8 +6065,8 @@ void moduleReleaseGIL(void) {
  *                               Notice, when this event fires, the given key
  *                               can not be retained, use RM_CreateStringFromString
  *                               instead.
- *  - REDISMODULE_NOTIFY_UNLINK: A special notification available only for modules,
- *                               indicates that the key was unlink from keyspace.
+ *  - REDISMODULE_NOTIFY_REMOVED: A special notification available only for modules,
+ *                                indicates that the key is removed from the keyspace.
```

Comment:
```suggestion
 *                                indicates that the key is about to be removed from the keyspace.
```
alternatively, maybe we can say `is being removed`, but i guess it's better to be explicit and state that it's not yet removed.